rancher-server/locals.tf

```diff
@@ -1,8 +1,9 @@
 locals {
-  name              = "rancher-demo"
+  name              = var.name
   rancher_version   = "v2.2.8"
```

Comment:
`rancher_version` should probably also be configurable for this to be a good module. Along with that, the helm repo that is being used should also be configurable

Reply:
parameterized chart and version.